### PR TITLE
Functional Tests - Test prefix in the invoice file name

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Test prefix in the invoice file name
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | TEST_PATH="functional/BO/orders/invoices/03_invoiceOptions/04_invoicePrefix.js" URL_FO=yourShopURL npm run specific-test

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16772)

### CODE REVIEW

**Review by SimonGrn — DISMISSED** [review]

**Review by boubkerbribri — DISMISSED** [review]
thanks @nesrineabdmouleh

File: tests/puppeteer/campaigns/functional/BO/orders/invoices/03_invoiceOptions/04_invoicePrefix.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+const InvoicesPage = require('@pages/BO/orders/invoices/index');
+const OrdersPage = require('@pages/BO/orders/index');
+const ViewOrderPage = require('@pages/BO/orders/view');
+// Importing data
+const {Statuses} = require('@data/demo/orders');
+
+let browser;
+let page;
+let fileName;
+const prefixToEdit = '#ABC';
```
Comment (PierreRambaud) [inline tests/puppeteer/campaigns/functional/BO/orders/invoices/03_invoiceOptions/04_invoicePrefix.js]: Should be in fixtures?

File: tests/puppeteer/pages/BO/orders/invoices/index.js [hunk 2]
```diff
@@ -30,6 +30,7 @@ module.exports = class Invoice extends BOBasePage {
     this.invoiceOptionEnableProductImage = `${this.invoiceOptionsForm} 
     label[for="form_invoice_options_enable_product_images_%ID"]`;
     this.saveInvoiceOptionsButton = `${this.invoiceOptionsForm} .btn.btn-primary`;
+    this.invoicePrefixInput = `${this.invoiceOptionsForm} #form_invoice_options_invoice_prefix_1`;
```
Comment (PierreRambaud) [inline tests/puppeteer/pages/BO/orders/invoices/index.js]:
```suggestion
    this.invoicePrefixInput = '#form_invoice_options_invoice_prefix_1';
```

Comment (PierreRambaud) [reply tests/puppeteer/pages/BO/orders/invoices/index.js]: duplicate line

File: tests/puppeteer/pages/BO/orders/invoices/index.js [hunk 3]
```diff
@@ -33,8 +33,10 @@ module.exports = class Invoice extends BOBasePage {
     this.legalFreeTextInput = '#form_invoice_options_legal_free_text_1';
     this.footerTextInput = '#form_invoice_options_footer_text_1';
     this.saveInvoiceOptionsButton = `${this.invoiceOptionsForm} .btn.btn-primary`;
+    this.invoicePrefixInput = `${this.invoiceOptionsForm} #form_invoice_options_invoice_prefix_1`;
     this.invoiceAddCurrentYear = `${this.invoiceOptionsForm} label[for="form_invoice_options_add_current_year_%ID"]`;
     this.optionYearPositionRadioButton = `${this.invoiceOptionsForm} #form_invoice_options_year_position_%ID`;
```
Comment (PierreRambaud) [inline tests/puppeteer/pages/BO/orders/invoices/index.js]:
```suggestion
    this.optionYearPositionRadioButton = '#form_invoice_options_year_position_%ID';
```

**Review by PierreRambaud — APPROVED** [review]

**Pull request merged**